GameEngine/GameEngine.cs

```diff
@@ -7,9 +7,9 @@ public class GameEngine
     {
         public void RunGame()
         {
-            var player = new LeastRecentCardAgent();
+            var player = new RandomAgent();
```

Comment:
These changes don't seem material to the rest of the change set.  If I understand correctly, they change the default player for the command line test.

Can we not add these changes to the change set for the PR?

Reply:
The card in trello had three items:  1) Make sure the algorithms look like they're doing the right thing.  2) Make sure the initial game state setup is winnable.  3) Add in multiplayer.  

After investigating the code base, I felt that it was working correctly.  However, it's not entirely obvious that this was the case.  None of the test cases really showcase that the agents function correctly (and it sounds like in at least some cases that's just going to be something we have to deal with because some of the agents are going to always exhaust the heap).  This change gets exercised in the GameEngineTests and it allows the test to produce additional output that indicates that the agent is able to sometimes win the game.  I was hoping that this would make it more obvious that the code base was functioning with respect to agent functionality without resorting to a new set of tests (some of which would always fail anyways).  

I'm not super attached to the change, but it does seem like we should give some hints or displays of agents actually winning games.  Maybe we add some suggestive comments someplace instead?  [Someone did mention something about a document that went along with this code base ... does the document talk about this sort of thing anywhere in it?]

Reply:
I don't have a preference either way here.

> Someone did mention something about a document that went along with this code base

I'm not aware of any documentation beyond the one in the repo (linked to in Slack).

Reply:
Okay, I'm going to pull it.  I don't think we're getting much out of it and I think any benefit that I might get out of it can be better served by the simple adversarial game repo that I have elsewhere.

Reply:
👍